ibis/expr/tests/test_datatypes.py

```diff
@@ -326,3 +326,8 @@ def test_timestamp_with_timezone_repr():
 def test_timestamp_with_timezone_str():
     ts = dt.Timestamp('UTC')
     assert str(ts) == "timestamp('UTC')"
+
+
+def test_time():
+    ts = dt.time
+    assert str(ts) == "time"
```

Comment:
Can you also add a test that checks the parsing of the time type , something like `assert dt.validate_type('time').equals(time)`